packages/expo-updates/android/src/androidTest/java/expo/modules/updates/logging/UpdatesLoggingTest.kt

```diff
@@ -125,41 +125,42 @@ class UpdatesLoggingTest {
     Assert.assertEquals("Message 2", UpdatesLogEntry.create(purgedLogs[0])?.message)
   }
 
-  @Test
-  fun testBridgeMethods() {
-    val asyncTestUtil = AsyncTestUtil()
-    val instrumentationContext = InstrumentationRegistry.getInstrumentation().context
-    val logger = UpdatesLogger(instrumentationContext)
-    logger.warn("Test message", UpdatesErrorCode.JSRuntimeError)
-    val entries = UpdatesModule.readLogEntries(
-      instrumentationContext,
-      1000L,
-    )
-    Assert.assertNotNull(entries)
-    Assert.assertEquals(1, entries.size)
-    val bundle = entries[0]
-    Assert.assertEquals("Test message", bundle.getString("message"))
-
-    var rejected = false
-    asyncTestUtil.asyncMethodRunning = true
-    UpdatesModule.clearLogEntries(instrumentationContext) { error ->
-      if (error != null) {
-        rejected = true
-        asyncTestUtil.asyncMethodRunning = false
-      }
-      asyncTestUtil.asyncMethodRunning = false
-    }
-    asyncTestUtil.waitForAsyncMethodToFinish("clearLogEntriesAsync timed out", 1000)
-    Assert.assertFalse(rejected)
-
-    val entries2 = UpdatesModule.readLogEntries(
-      instrumentationContext,
-      1000L
-    )
-    asyncTestUtil.waitForAsyncMethodToFinish("readLogEntriesAsync timed out", 1000000)
-    Assert.assertNotNull(entries2)
-    Assert.assertEquals(0, entries2.size)
-  }
+  // TODO: Reenale this after upgrading react-native to 0.73
```

Comment:
Synced with @douglowder on Slack about this, Will will investigate after this lands on `main`